test/integration/kill/test_kill.py

```diff
@@ -0,0 +1,187 @@
+import os
+import re
+
+import pytest
+import ttapp
+
+import utils
+
+tarantool_major_version, tarantool_minor_version = utils.get_tarantool_version()
+
+skip_cluster_cond = tarantool_major_version < 3
+skip_cluster_reason = "skip cluster instances test for Tarantool < 3"
+
+# Values to be used to parametrize input at confirmation prompt.
+confirmation_input_params = [
```

Comment:
And why isn't enough just one test that checks all these options, why do you need to repeat the same for each test?

Reply:
> Note: On CI the duration of such a sleep is very flaky.

Agree. I'm working to drop it.

Reply:
> And why isn't enough just one test that checks all these options, why do you need to repeat the same for each test?

There are 2 parameterized tests: against multi-instance app and against cluster app. Each of them checks various combinations of "input" parameters, such as initially running instances and targets kill is requested for. What does mean "for each test"? Please, clarify.

Reply:
I see this variable mentioned 11 times. In my mind, this functionality should be checked once in a corresponding test that checks the logic of entering an answer, not its result.

Reply:
> I see this variable mentioned 11 times. In my mind, this functionality should be checked once in a corresponding test

`tt` implements multiple commands and some of them in some circumstances require user confirmation. This PR introduces tests that check various scenarios with 3 commands: restart, stop, kill. Various scenarios here means various combinations of different application type (simple app, multi-instance app, cluster app) with various user answers. These scenarios are distributed among tests based on command, then on application type. I used this kind of distribution based on current test layout that groups tests by commands. Do you suggest to rearrange these tests so single test checks user interaction against all command and application types? If so then what's the benefit of such arrangement? It may eliminate multiple references to this parameter set at a price of another parameter set duplication (like set representing collection of commands or set representing collection of application types).

> checks the logic of entering an answer, not its result

Sorry, still don't understand. What does it mean? What is "the logic of entering an answer" and how it should be checked? And what does mean "not its result"? What is the result? Is it the way execution goes after confirmation or discarding? If so and it should not be used, then how "the logic of entering an answer" should be checked?

It seems there is some key difference in conception regarding these tests, so I just can't understand your point. Would you please explain in other words.

My vision is scenario-based:
There are tons of scenarios `tt` might be used and we need to cover as much as we can with tests (considering CI limitations). This PR introduces tests that cover considerable amount of scenarios for 6 commands: start/stop/restart/kill/logrotate/clean. Arrangement of tests based on current hierarchy, i.e. at first level tests are grouped by the `tt` command they check.